benches/README.md

```diff
@@ -0,0 +1,72 @@
+# Rust Bitcoin Benchmarks
+
+Criterion based benchmarks for `rust-bitcoin`.
+
+## Running the benchmarks
+
+Examples below are run from within the crates folder `benches/`, if running from the repo root pass in `--manifest-path benches/Cargo.toml`.
+
+Run all benchmarks in this crate:
+
+```bash
+cargo bench
```

Comment:
If I remember, this requires the nightly toolchain which would be worth a mention.  Or you could do `cargo +nightly bench` here I think.

Reply:
Yeah, as long as something's in the top-level `workspace.exclude` we can require pretty-much anything.

I do think it's possible to get an old version of criterion that works with 1.74, but agree that it's not particularly urgent.